src/main.js

```diff
@@ -120,6 +120,22 @@ if ('serviceWorker' in window.navigator) {
     });
 }
 
+window.setTimeout(function () {
```

Comment:
I want to avoid this.  A better way for us to do this is to take the HTML you have below, and insert it into a `<div id="iframe-warning">..</div>` inside the spinner container, [here](
https://github.com/mozilla/brackets/blob/master/src/index.html#L118-L120).  You can add some CSS to the [inline style block](https://github.com/mozilla/brackets/blob/master/src/index.html#L31) to hide this `div` by default.  Then, when your JS runs, if we enter the `if` block you have on 129`, we can show the error (e.g., add/change the CSS class on the div) and skip loading Bramble.